srml/finality-tracker/Cargo.toml

```diff
@@ -16,7 +16,7 @@ impl-trait-for-tuples = "0.1.1"
 
 [dev-dependencies]
 primitives = { package = "substrate-primitives",  path = "../../core/primitives", default-features = false }
-runtime-io = { package = "sr-io", path = "../../core/sr-io", default-features = false }
+runtime-io = { package = "sr-io", path = "../../core/sr-io" }
```

Comment:
Why don't we want `default-features` here?

Reply:
Not sure.

Reply:
Hm, this actually sounds like it might be the root problem for [this](https://github.com/paritytech/substrate/pull/3556#discussion_r329024324)

Reply:
Good catch, thanks!